infra/recipes/default.rb

```diff
@@ -0,0 +1,51 @@
+USER = "vagrant"
+WORKSPACE = "/home/vagrant/workspace"
+PATH = "/opt/rbenv/shims:/opt/rbenv/bin:/opt/rbenv/plugins/ruby_build/bin:#{ENV['PATH']}";
```

Comment:
Line is too long. [90/80]<br>Do not use semicolons to terminate expressions.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.